cirq-core/cirq/study/sweeps.py

```diff
@@ -372,7 +372,7 @@ def __len__(self) -> int:
     def param_tuples(self) -> Iterator[Params]:
         iters = [sweep.param_tuples() for sweep in self.sweeps]
         for values in zip(*iters):
-            yield sum(values, ())
+            yield tuple(itertools.chain.from_iterable(values))
```

Comment:
OOC, how does the performance of this compare to a plain comprehension?
```suggestion
            yield tuple(param for params in value for param in params)
```

Reply:
Cool, thanks for checking!